projects/practical-scheme.net/gauche/package.yml

```diff
@@ -0,0 +1,57 @@
+distributable:
+  url: https://github.com/shirok/Gauche/releases/download/{{version.tag}}/Gauche-{{version}}.tgz
+  strip-components: 1
+
+versions:
+  github: shirok/Gauche
+  strip: /^release/
+
+dependencies:
+  curl.se/ca-certs: '>=2023'
+  github.com/Mbed-TLS/mbedtls: ^3.5
+  github.com/besser82/libxcrypt: ^4.4
+  zlib.net: ^1.3
+
+runtime:
+  env:
+    GAUCHE_LOAD_PATH: ${{prefix}}/etc/GAUCHE_LOAD_PATH:{{prefix}}/etc/GAUCHE_LOAD_PATH/gauche:$GAUCHE_LOAD_PATH
+    GAUCHE_DYNLOAD_PATH: ${{prefix}}/etc/GAUCHE_DYNLOAD_PATH:$GAUCHE_DYNLOAD_PATH
+
+build:
+  script:
+    - ./configure $ARGS
+    - make --jobs {{hw.concurrency}}
+    - make --jobs {{hw.concurrency}} install
+    - run: sed -i 's|#!{{prefix}}/bin/gosh|#!/usr/bin/env gosh|g' gauche-cesconv gauche-install gauche-package
```

Comment:
should be able to just run `fix-shebangs.ts gauche-cesconv gauche-install gauche-package` which currently does just that.